content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+=============
+Configuration
+=============
+
+The following document goes through all of the configurations and settings for the *Fleet*
+application, including settings, manufacturers, models, and categories.
+
+Settings
+========
+
+To access the settings menu, go to :menuselection:`Fleet application --> Configuration -->
+Settings`. There are only two settings that need configuration, the vehicle contract alert, and
+a new vehicle request.
+
+End Date Contract Alert
+-----------------------
+
+To help ensure no contracts lapse or vehicles are left without contracted drivers, Odoo has an
+alert in place to let the person responsible for the vehicle contract know when a contract is abut
+to expire. Enter the number of days before a contract is going to expire that an alert email will be
+sent.
```

Comment:
```suggestion
To help ensure no contracts lapse or so no vehicles are left without contracted drivers, Odoo has an
alert in place to let the person responsible for the vehicle contract, know when a contract is about
to expire. Enter the number of days before a contract is going to expire, so that an alert email will be
sent.
```

This paragraph needs to be reworded. Last sentence doesn't make sense and first line is a bit confusing